C/prefix.h

```diff
@@ -1,3 +1,3 @@
 #ifndef SIMPLICITY_PREFIX
-#define SIMPLICITY_PREFIX "Simplicity-Draft"
```

Comment:
Oops, this prefix is incorrect.  It should be "Simplicity" not "Simplicity-1" (something that I was toying with in an earlier revision).

I'm surprised this didn't tickle any test error, but it turns out that since doing `precomputation.h`, this isn't actually used anymore.  I'll followup by simply deleting this.